canonical-web-and-design/multipass.run
# Add the official logo and favicon

## Done
Add the official logo and favicon

## QA
- Run `./run`
- Open http://0.0.0.0:8026
- Check the logo in the navigation is multipass
- Check the favicon is multipass

Fixes https://github.com/canonical-web-and-design/multipass.run/issues/23
